### PR TITLE
Payment section added

### LINKED ISSUE #20: Add bill to User profile
The user profile needs to have a bill stating the paid amount, and the due amount for his bookings. This should have entries only for the confirmed bookings.

### PR DESCRIPTION
**Pull request by yashikabaan**
Fixes #20

### CODE REVIEW

**Comment by rishi-s8** [comment]
@yashikabaan this isn't working. Clicking on the Awaiting Requests in admin mode gives: ```MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'b.Payment' in 'field list'")```

File: templates/profile.html [hunk 1]
```diff
@@ -16,6 +16,7 @@ <h1> User Profile </h1>
       <th>SUPERVISOR NAME</th>
       <th>SUPERVISOR EMAIL</th>
       <th>REQUEST STATUS</th>
+      <th>PAYMENT STATUS</th>
```
Comment (rishi-s8) [inline templates/profile.html:20]: This should also be there on accessing the `user/<userId>` route.

File: templates/profile.html [hunk 2]
```diff
@@ -30,6 +31,11 @@ <h1> User Profile </h1>
           <a href="/confirm_reschedule/{{booking.BookingID}}"><input type='submit' class="btn btn-primary" value="Accept/Reject"></a>
         {% endif %}
       </td>
+      {% if booking.Payment == 1 %}
+      <td>paid</td>
```
Comment (rishi-s8) [inline templates/profile.html]: P should be upper case.

File: templates/requests.html [hunk 3]
```diff
@@ -25,6 +26,11 @@ <h1>Update Request Status</h1>
       <td>{{req.SName}}</td>
       <td>{{req.SEmail}}</td>
       <td><a href="/update_booking/{{req.BookingID}}"><input type = 'submit' class = "btn btn-primary" value = "Update"></a></td>
+      {% if req.Payment == 1 %}
+      <td><a href="/update_payment/{{req.BookingID}}"><input type = 'submit' class = "btn btn-primary" value = "paid"></a></td>
```
Comment (rishi-s8) [inline templates/requests.html]: 'P'aid

**Comment by Nivedit24** [comment]
Also switch the order of the payment and request status columns. paid shoud be in uppercase. Do we need 2 buttons ?

**Comment by rishi-s8** [comment]
@Nivedit24 yes, I wasn't able to see how this is actually working from the code. That is why I wanted to test. 
@yashikabaan you did not create the new sqldump probably.

**Review by rishi-s8 — CHANGES_REQUESTED** [review]

**Review by rishi-s8 — APPROVED** [review]

**Comment by rishi-s8** [comment]
Fixed the working of the code. Works good now.

**Pull request merged**